vercel/resource_project.go

```diff
@@ -176,21 +175,14 @@ At this time you cannot use a Vercel Project resource with in-line ` + "`environ
 			"vercel_authentication": schema.SingleNestedAttribute{
 				Description: "Ensures visitors to your Preview Deployments are logged into Vercel and have a minimum of Viewer access on your team.",
 				Optional:    true,
-				Computed:    true,
```

Comment:
Computed was added just because this has a default value when a new project is created. Need to make sure you can omit this field and it works.